packages/nimble-components/src/table-column/text/cell-view/template.ts

```diff
@@ -0,0 +1,19 @@
+import { html, ref } from '@microsoft/fast-element';
+
+import type { TableColumnTextCellView } from '.';
+
+export const template = html<TableColumnTextCellView>`
+    <span
+        ${ref('textSpan')}
+        class="${x => (typeof x.cellRecord.value === 'string' ? '' : 'placeholder')}"
+        @mouseover="${x => {
+        x.isHoveredWithOverflow = !!x.content && x.textSpan.offsetWidth < x.textSpan.scrollWidth;
```

Comment:
nit:
From this [example impl](isHoveredWithOverflow) the parts I like are keeping the template logic simple and that the boolean value, i.e. `isHoveredWithOverflow` is based only on the overflow calculation (the current name is more like isValidContentAndHasOverflow).

Could do the name change to `isValidContentAndHasOverflow` or the helper methods or neither. w/e feels easiest to follow.

Reply:
Your link is broken so wasn't sure which example/psuedocode you were linking to. Updated the property name to `isValidContentAndHasOverflow` though.